ckan/cli/file.py

```diff
@@ -0,0 +1,567 @@
+
+
+@stats.command("overview")
+@storage_option
+def stats_overview(storage_name: str | None):
+    """General information about storage usage.
+
+    Computed using registered files. If storage contains files without
+    corresponding DB record, they won't be added to these numbers.
+    """
+    storage_name = storage_name or config["ckan.files.default_storages.default"]
+    try:
+        files.get_storage(storage_name)
+    except files.exc.UnknownStorageError as err:
+        error_shout(f"Storage {storage_name} is not configured")
+        raise click.Abort from err
+
+    stmt = sa.select(
+        sa.func.sum(model.File.size),
+        sa.func.count(model.File.id),
+        sa.func.max(model.File.ctime),
+        sa.func.min(model.File.ctime),
+    ).where(model.File.storage == storage_name)
+    row = model.Session.execute(stmt).fetchone()
+    size, count, newest, oldest = row if row else (0, 0, _now(), _now())
+
+    if not count:
+        error_shout(f"Storage {storage_name} is empty")
+        raise click.Abort
+
+    click.secho(f"Number of files: {click.style(count, bold=True)}")
+    click.secho(
+        f"Used space: {click.style(fk.humanize_filesize(size), bold=True)}",
+    )
+    click.secho(
+        "Newest file created at: "
+        + f"{click.style(format_datetime(newest), bold=True)} "
+        + f"({format_timedelta(newest - _now(), add_direction=True)})",
+    )
+    click.secho(
+        "Oldest file created at: "
+        + f"{click.style(format_datetime(oldest), bold=True)} "
+        + f"({format_timedelta(oldest - _now(), add_direction=True)})",
+    )
+
+
+@stats.command("types")
+@storage_option
+def stats_types(storage_name: str | None):
+    """Files distribution by MIME type."""
+    storage_name = storage_name or config["ckan.files.default_storages.default"]
+    stmt = (
+        sa.select(
+            model.File.content_type,
+            sa.func.count(model.File.content_type).label("count"),
+        )
+        .where(model.File.storage == storage_name)
+        .group_by(model.File.content_type)
+        .order_by(model.File.content_type)
+    )
+
+    total = model.Session.scalar(sa.select(sa.func.sum(stmt.c.count)))
+    click.secho(
+        f"Storage {click.style(storage_name, bold=True)} contains "
+        + f"{click.style(total, bold=True)} files",
+    )
+    for content_type, count in model.Session.execute(stmt):
+        click.secho(f"\t{content_type}: {click.style(count, bold=True)}")
+
+
+@stats.command("owner")
+@storage_option
+@click.option(
+    "-v",
+    "--verbose",
+    is_flag=True,
+    help="Show distribution for every owner ID",
+)
+def stats_owner(storage_name: str | None, verbose: bool):
+    """Files distribution by owner."""
+    storage_name = storage_name or config["ckan.files.default_storages.default"]
+    owner_col = (
+        sa.func.concat(model.Owner.owner_type, " ", model.Owner.owner_id)
+        if verbose
+        else sa.func.concat(model.Owner.owner_type, "")
```

Comment:
It looks like `.label()` works on orm attributes
```suggestion
        else model.Owner.owner_type
```